File: solutions/java/src/stackoverflow/entities/Answer.java
package stackoverflow.entities;

import java.util.UUID;

public class Answer extends Post {
    private boolean isAccepted = false;

    public Answer(String body, User author) {
        super(UUID.randomUUID().toString(), body, author);
    }

    public void setAccepted(boolean accepted) {
        isAccepted = accepted;
    }

    public boolean isAccepted() { return isAccepted; }
}

File: solutions/csharp/stackoverflow/Models/Answer.cs
class Answer : Post
{
    private bool isAccepted = false;

    public Answer(string body, User author)
        : base(Guid.NewGuid().ToString(), body, author)
    {
    }

    public void SetAccepted(bool accepted)
    {
        isAccepted = accepted;
    }

    public bool IsAcceptedAnswer() { return isAccepted; }
}